src/backend_model.h

```diff
@@ -123,8 +124,8 @@ class TritonModel : public Model {
   std::shared_ptr<TritonBackend> backend_;
 
   // The model instances for this model.
```

Comment:
Unrelated to this ticket, but general quality-of-life: it'd be great to clarify in this comment (or in AddInstance) what passive means  here. No need to update it here, but if you're working on these files, would be awesome to clarify it in this or a future PR.

Reply:
Updated the comment (hopefully) clarifying the storage container and the definition of the passive instance groups.